rest/types/config.go

```diff
@@ -0,0 +1,8 @@
+package types
+
+// Config is the in memory version of the local daemon.yaml file.
+type Config struct {
```

Comment:
Perhaps this should be called `DaemonConfig`? I know it stammers with the `config` package's type but since this is a public type, if we ever decide that we need more types of config and this name is too generic, then every project will have to refactor their code. (Like with `Server` for the past few weeks :p)

Reply:
Thanks will fix! Previously I had this type together with the daemon's `DaemonConfig` internal type but when implementing the hook I have moved it into the public types package but haven't changed it's name.

Reply:
Fixed